packages/cli/src/utils/config.ts

```diff
@@ -0,0 +1,90 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import * as clack from '@clack/prompts'
+import chalk from 'chalk'
+import { BuildTool } from '../types'
+import { abortIfCancelled, getNextRouter } from './utils'
+import { buildTools } from './constants'
+import { modifyConfigFile } from './modify_config'
+
+export function detectBuildTool(): BuildTool | null {
+  /**
+   * Detect build tool by checking names of config files if they exist
+   */
+  for (const buildTool of buildTools) {
+    for (const fileName of buildTool.possibleFileNames) {
+      /**
+       * Check for all extentions
+       */
+      if (fs.existsSync(path.join(process.cwd(), fileName))) {
+        let xbuildTool = { ...buildTool, configFilePath: fileName }
```

Comment:
what does `xbuildTool` mean in this case, possibly switch to a better name, like `currentBuildTool`. Also switch from let to const

Reply:
Thanks for pointing out I forgot to change this.